teps/0079-tekton-catalog-support-tiers.md

```diff
@@ -100,14 +128,460 @@ requirements, and provenance attestations can be made to meet software supply ch
+Official Catalog:
+
+1. The resource MUST comply with the contract defined in
+   [TEP-0003: Tekton Catalog Organization][tep-0003-org].
+2. The resource MUST not define an OWNER file, given that it is owned
+   and maintained by Tekton Catalog Maintainers who are defined at the
+   root of the Catalog. The ownership responsibilities are described
+   [below](#responsibilities).
+3. The resource MUST have automated testing using Tekton for dogfooding.
+   If there are any failures, they MUST be resolved as soon as possible;
+   the SLO is one week. The automated testing is discussed further
+   [below](#automated-testing-and-dogfooding-1).
+4. The resource MUST be scanned for common vulnerabilities and exposures.
+   If any issues are discovered, they MUST be patched or disclosed as soon as
+   possible; the SLO is one week. The scanning for CVEs is discussed further
+   [below](#image-scanning-for-common-vulnerabilities-and-exposures-cves-1).
+5. The resource MUST support verification as proposed in
+   [TEP-0091: Verified Remote Resources][tep-0091]. For now, it MUST be
+   published to a public OCI registry as a [Tekton Bundle][bundle] and
+   signed by Tekton. In the future, it MUST be updated to support accepted
+   proposal of TEP-0091 depending on the direction of
+   [TEP-0060: Remote Resource Resolution][tep-0060].
+   The verification is discussed further [below](#verified-remote-resources-1).
+6. The resource MUST be well documented with all configuration options described
+   and a working example provided.
+7. The resource MUST follow and demonstrate best practices, such as the
+   [`Task` authoring recommendations][task-authoring-recommendations].
+
+#### Promotion from Community Catalog to Official Catalog
+
+Promoting a resource from the Community Catalog to the Official Catalog 
+should be an exception, not the norm. The set of resources in the 
+Official Catalog should relatively small so that it's sustainable for the 
+Tekton Catalog Owners to maintain. 
+
+A resource in the Community Catalog may get promoted to the Official Catalog 
+if:
+* _Quality_: It meets the [requirements of the Official Catalog](#requirements-1).
+* _Bandwidth_: The [Tekton Catalog Maintainers][catalog-owners] approve its
+  promotion based on its usefulness to the community and existing bandwidth
+  to maintain the resource.
+
+If a resource is promoted to the Official Catalog:
+* Its minor version MUST be bumped during the migration.
+  For example, if we want to bump *git-clone* `Task` whose latest version 
+  is 0.5 in the Community Catalog, we would add version 0.6 to the Official
+  Catalog. This ensures that there won't be multiple resources with the same 
+  name and version in the cluster.
+* It can be [deprecated][deprecation] in the Community Catalog. A resource 
+  is deprecated through the `tekton.dev/deprecated: "true"` annotation.
+
+We will provide tooling, through [Caitlin][caitlin], to make it easier for
```

Comment:
total nit: i think the tool is called `catlin`? but i see it referred to as `caitlin` a few times (or maybe the name changed and i missed it XD)

Reply:
🤣 ah it's `catlin` from catalog lint, been calling it `caitlin` all this time and thought it was a cute name! 

fixed it, thanks @bobcatfish 😀